.github/workflows/scanbuild_static-analysis.yml

```diff
@@ -138,15 +138,14 @@ jobs:
       working-directory: ./racket/src
       run: |
         ./configure --enable-cgcdefault --prefix=/usr
+        # nproc?
         export cpus=$(grep -c ^processor /proc/cpuinfo)
```

Comment:
@pmatos 

Should we use `nproc` instead of grepping `/proc/cpuinfo`? https://www.flamingspork.com/blog/2020/11/25/why-you-should-use-nproc-and-not-grep-proc-cpuinfo/ seems to indicate that `nproc` is better.

Also, sometimes we use the value `cpus`. Sometimes we use the value `cpus+1`. When should we use one or another?

Reply:
OK, I think it doesn't matter for default runners. They produce the same number. It's unclear what's the result for the self-hosted runner.